Classes/Traits/InjectUriBuilderTrait.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the package jweiland/jwtools2.
+ * For the full copyright and license information, please read the
+ * LICENSE file that was distributed with this source code.
+ */
+
+namespace JWeiland\Jwtools2\Traits;
+
+use TYPO3\CMS\Extbase\Mvc\Web\Routing\UriBuilder;
+
+/**
+ * Trait to inject UriBuilder
+ */
+trait InjectUriBuilderTrait
+{
+    protected ?UriBuilder $uriBuilder = null;
```

Comment:
Why nullable. It's part of the initialization process.

Reply:
![image](https://github.com/jweiland-net/jwtools2/assets/144037456/9c110140-d221-4468-a85d-2388db633757)

https://docs.typo3.org/m/typo3/reference-coreapi/main/en-us/ApiOverview/DependencyInjection/Index.html#method-injection

@sfroemkenjw I had your same thought. As it is mentioned in the official documentation I keep it like this